convert-hf-to-gguf.py

```diff
@@ -857,25 +857,20 @@ def parse_args() -> argparse.Namespace:
     print(f'Error: {args.model} is not a directory', file=sys.stderr)
     sys.exit(1)
 
-# possible tensor data types
-#   ftype == 0 -> float32
-#   ftype == 1 -> float16
-
-# map from ftype to string
 ftype_str = ["f32", "f16"]
```

Comment:
I would prefer if this became an explicit map from strings to enum values, like this:
```python
ftype = {
    "f32": GGMLQuantizationType.F32,
    "f16": GGMLQuantizationType.F16,
}[args.ftype]
```
And remove the `ftype = args.ftype` line above, and use `args.ftype` in the filename.